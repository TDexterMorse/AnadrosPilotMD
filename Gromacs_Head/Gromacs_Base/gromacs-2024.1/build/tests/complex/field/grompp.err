                      :-) GROMACS - gmx grompp, 2024.1 (-:

Executable:   /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/bin/gmx
Data prefix:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1 (source tree)
Working dir:  /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/complex/field
Command line:
  gmx grompp -f /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/grompp.mdp -c /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/conf -r /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/conf -p /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/topol -maxwarn 10

Ignoring obsolete mdp entry 'optimize_fft'
Ignoring obsolete mdp entry 'rlistlong'
Ignoring obsolete mdp entry 'nstcalclr'
Ignoring obsolete mdp entry 'gb_algorithm'
Ignoring obsolete mdp entry 'nstgbradii'
Ignoring obsolete mdp entry 'rgbradii'
Ignoring obsolete mdp entry 'gb-epsilon-solvent'
Ignoring obsolete mdp entry 'gb_saltconc'
Ignoring obsolete mdp entry 'gb-obc-alpha'
Ignoring obsolete mdp entry 'gb-obc-beta'
Ignoring obsolete mdp entry 'gb-obc-gamma'
Ignoring obsolete mdp entry 'gb-dielectric-offset'
Ignoring obsolete mdp entry 'sa-algorithm'
Ignoring obsolete mdp entry 'sa-surface-tension'
Ignoring obsolete mdp entry 'ns_type'

NOTE 1 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/grompp.mdp]:
  With Verlet lists the optimal nstlist is >= 10, with GPUs >= 20. Note
  that with the Verlet scheme, nstlist has no effect on the accuracy of
  your simulation.

Generating 1-4 interactions: fudge = 0.5
Number of degrees of freedom in T-Coupling group rest is 1293.00
The integrator does not provide a ensemble temperature, there is no system ensemble temperature

NOTE 2 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/grompp.mdp]:
  NVE simulation: will use the initial temperature of 212.778 K for
  determining the Verlet buffer size


NOTE 3 [file /home/ubuntu/W4GROMACSPIPE/Gromacs_Head/Gromacs_Base/gromacs-2024.1/build/tests/regressiontests-2024.1/complex/field/grompp.mdp]:
  You are using a plain Coulomb cut-off, which might produce artifacts.
  You might want to consider using PME electrostatics.



There were 3 NOTEs

Thanx for Using GROMACS - Have a Nice Day

