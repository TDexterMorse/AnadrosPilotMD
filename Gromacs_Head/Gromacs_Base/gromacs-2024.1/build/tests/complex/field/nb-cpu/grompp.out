Generated 330891 of the 330891 non-bonded parameter combinations

Generated 330891 of the 330891 1-4 parameter combinations

Excluding 2 bonded neighbours molecule type 'SOL'
Analysing residue names:
There are:   216      Water residues

Determining Verlet buffer for a tolerance of 0.005 kJ/mol/ps at 212.778 K

Calculated rlist for 1x1 atom pair-list as 0.921 nm, buffer size 0.021 nm

Set rlist, assuming 4x4 atom pair-list, to 0.916 nm, buffer size 0.016 nm

Note that mdrun will redetermine rlist based on the actual pair-list setup

This run will generate roughly 0 Mb of data
